packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,87 @@ section: get-started
 import './get-started.css';
 import { Divider } from '@patternfly/react-core';
 
+## PatternFly 5.2 
+
+PatternFly 5.2 is now live! Here are the highlights from this release. 
+
+### Major release preparation
+
+The next major version of PatternFly (v6) doesn't come out until later this year, but our team has started preparing some of its most significant updates.
+
+Our major development work from the past quarter lays the groundwork for our ***upcoming v6 alpha release***, which is slotted for Q1 2024.
+
+#### Design token support 
+
+To prepare for PatternFly's upcoming addition of a new visual theme, called "Penta", we added support for design tokens to many of our components. This support has been enabled in both the visual design of each component, as well as in their code implementation. 
+
+#### PatternFly React seed
+
+We readied our [React seed codebase](https://github.com/patternfly/patternfly-react-seed) for the v6 alpha release, so that any web applications built on PatternFly will have access to our Penta theme at the same time as our React and HTML consumers.
```

Comment:
I don't know that this is relevant for this release.  @nicolethoen isn't that work in v6 branch?

Reply:
it is in a branch, and I'd agree. Notes on the PF-react-seed are not quite relevant to this release.